### PR TITLE
Runtime code for install and start

### PR DESCRIPTION
**Pull request by davidkel**
Signed-off-by: Dave Kelsey <d_kelsey@uk.ibm.com>

### CODE REVIEW

File: packages/composer-admin/changelog.txt [hunk 1]
```diff
@@ -11,6 +11,9 @@
 #
 # Note that the latest public API is documented using JSDocs and is available in api.txt.
 #
+Version 0.9.3 {3af411e8da53bb013ab9718ed5980c20} 2017-07-17
```
Comment (sstone1) [inline packages/composer-admin/changelog.txt]: Needs changing for 0.10.0.

File: packages/composer-common/lib/connection.js [hunk 2]
```diff
@@ -88,6 +88,32 @@ class Connection extends EventEmitter {
     }
 
     /**
+     * Install the hyperledger composer runtime.
```
Comment (sstone1) [inline packages/composer-common/lib/connection.js]: Case needs to be fixed - `Hyperledger Composer`.

File: packages/composer-admin/changelog.txt (hunk 1 above)
Comment (davidkel) [inline packages/composer-admin/changelog.txt]: Think I need to version bump first before I can change this.

File: packages/composer-common/lib/connection.js (hunk 2 above)
Comment (davidkel) [inline packages/composer-common/lib/connection.js]: Good point, Thanks

**Comment by sstone1** [comment]
Looks great, good work @davidkel.

**Pull request merged**